src/flows/shared/BucketSelector/index.tsx

```diff
@@ -44,6 +65,8 @@ const BucketSelector: FC = () => {
   if (loading === RemoteDataState.Done && buckets.length) {
     menuItems = (
       <>
+        <CreateBucketDropdownItem onClick={handleCreateNewBucket} />
```

Comment:
why is this something special and not another Dropdown.Item?

Reply:
It had a lot of things going on specifically for it, and it made the parent component feel messy so I isolated it